src/main/java/nl/quintor/studybits/indy/wrapper/Verifier.java

```diff
@@ -56,14 +56,14 @@ private void validateResult(boolean valid, String message) throws IndyWrapperExc
         }));
     }
 
-    private void validateProofEncodings(Proof proof) {
+    private void validateProofEncodings(Proof proof, String theirDid) {
```

Comment:
This code is also duplicated in the utility class, so let's remove this instance.